### PR TITLE
Adding the min version on Stats calculator

### PR DESCRIPTION
**Pull request by rtamiri**

### CODE REVIEW

File: src/main/scala/com/clarify/stats/slim/StatsCalculator.scala [hunk 1]
```diff
@@ -0,0 +1,152 @@
+package com.clarify.stats.slim
```
Comment (roman-y) [inline src/main/scala/com/clarify/stats/slim/StatsCalculator.scala:1]: I don't like its path src/main/scala/com/clarify/stats/slim/StatsCalculator.scala
it's inconsistent with src/main/scala/com/clarify/stats/v1/StatsCalculator.scala
I'd say rename this file to StatsCalculatorSlim.scala and add it to src/main/scala/com/clarify/stats/v1/

**Review by roman-y — CHANGES_REQUESTED** [review]

File: src/main/scala/com/clarify/stats/slim/StatsCalculator.scala [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+import java.util
+
+import com.clarify.Helpers
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{DataFrame, Row, SQLContext}
+
+object StatsCalculator {
```
[the remark this answers is not in the document]
Comment (rtamiri) [reply src/main/scala/com/clarify/stats/slim/StatsCalculator.scala:11]: fixed

File: src/main/scala/com/clarify/stats/slim/StatsCalculator.scala (hunk 1 above)
Comment (rtamiri) [reply src/main/scala/com/clarify/stats/slim/StatsCalculator.scala:1]: The folder path should contain one Scala file. I can create another new folder

Comment (roman-y) [reply src/main/scala/com/clarify/stats/slim/StatsCalculator.scala:1]: "The folder path should contain one Scala file" - why?

Comment (rtamiri) [reply src/main/scala/com/clarify/stats/slim/StatsCalculator.scala]: I mean for consistency with other scripts in the repo.

Comment (roman-y) [reply src/main/scala/com/clarify/stats/slim/StatsCalculator.scala]: I don't know of such rule and I don't see that it's being followed eg https://github.com/clarifyhealth/spark-sparse-vector/tree/master/src/main/scala/com/clarify/prediction_explainer
So still unclear as to what the reason is. If you want to be consistent why are you creating folder `slim` on the same level where we create version folders like `v1`?

Comment (rtamiri) [reply src/main/scala/com/clarify/stats/slim/StatsCalculator.scala]: fixed

**Review by roman-y — APPROVED** [review]

**Pull request merged**